omics/analyzer/omics-run-analyzer

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/env python3
+"""
+Generate statistics for a completed HealthOmics workflow run
+
+Usage: omics-run-analyzer [<runId>...]
+                          [--profile=<profile>]
+                          [--region=<region>]
+                          [--time=<interval>]
+                          [--show]
+                          [--timeline]
+                          [--file=<path>]
+                          [--out=<path>]
+                          [--help]
+
+Options:
+ -p, --profile=<profile>  AWS profile
+ -r, --region=<region>    AWS region
+ -t, --time=<interval>    Select runs over a time interval [default: 1day]
+ -s, --show               Show run resources with no post-processing (JSON)
+ -T, --timeline           Show workflow run timeline
+ -f, --file=<path>        Load input from file
+ -o, --out=<path>         Write output to file
+ -h, --help               Show help text
+
+Examples:
+ # Show workflow runs that were running in the last 5 days
+ omics-run-analyzer -t5d
+ # Retrieve and analyze a specific workflow run by ID
+ omics-run-analyzer 1234567 -o run-1234567.csv
+ # Retrieve and analyze a specific workflow run by ID and UUID
+ omics-run-analyzer 2345678:12345678-1234-5678-9012-123456789012
+ # Output workflow run and tasks in JSON format
+ omics-run-analyzer 1234567 -s -o run-1234567.json
+"""
+import csv
+import datetime
+import json
+import os
+import re
+import sys
+
+import boto3
+import dateutil
+import docopt
+
+exename = os.path.basename(sys.argv[0])
+omics_log_group = "/aws/omics/WorkflowLog"
+omics_service_code = "AmazonOmics"
+pricing_aws_region = "us-east-1"  # Pricing service endpoint
+
+
+def die(msg):
+    """Show error message and terminate"""
+    exit(f"{exename}: {msg}")
+
+
+def parse_time_str(s, utc=True):
+    """Parse time string"""
+    tz = datetime.timezone.utc
+    return dateutil.parser.parse(s).replace(tzinfo=tz) if s else None
+
+
+def parse_time_delta(s):
+    """Parse time delta string"""
+    m = re.match(r"(\d+)\s*(m|min|minutes?|h|hours?|d|days?|w|weeks?)$", s)
```

Comment:
can this go out to months or years?

Reply:
Yes. I've intentionally not included months here, because month length varies. Could include years here, but the search for manifest could take a very long time if the customer has generated a large number of runs.